Source/Core/DolphinQt/GameList/GameFile.cpp

```diff
@@ -26,23 +25,48 @@
 #include "DolphinQt/Utils/Resources.h"
 #include "DolphinQt/Utils/Utils.h"
 
-static const u32 CACHE_REVISION = 0x006;
+static const u32 CACHE_REVISION = 0x007;
 static const u32 DATASTREAM_REVISION = 15; // Introduced in Qt 5.2
 
-static QStringList VectorToStringList(std::vector<std::string> vec, bool trim = false)
+static QMap<IVolume::ELanguage, QString> ConvertLocalizedStrings(std::map<IVolume::ELanguage, std::string> strings)
```

Comment:
This function only existed before because Qt can't magically convert `std::string` to a `QString`, but now that you're just using a `std::map` with the same template args as the `QMap`, you can delete this function and just use the [conversion initializer of `QMap`](http://doc.qt.io/qt-5/qmap.html#QMap-3).